### ISSUE TITLE
error add trip

### ISSUE THREAD

**Issue by lewandowskit** [tributeee/laravel-gpx-trip-visualizer#1]
Hello i found your code ŚCIĄGŁEM and how to add a tour in GPX format then I get information that not a valid file

**Comment by WoodwardT** [comment]
Yeah I have the same problem as above. When trying to upload a .gpx or .xml it says it's an invalid format and you must upload in .gpx or .xml format.
